src/services/cohortCreation/index.ts

```diff
@@ -40,9 +40,9 @@ const getPatientsFromDocuments = memoize(
   async (documentQuery: string, patientFilter: string): Promise<(string | undefined)[]> => {
     const response = await adminApiFhir.get<FHIR_API_Response<IDocumentReference>>(documentQuery)
     const documents = getApiResponseResources(response)
-    const patientIds = documents
-      ?.map((document) => last(document.subject?.reference?.split('/')))
-      .filter((patientId) => !!patientId)
+    const patientIds = uniq(
+      documents?.map((document) => last(document.subject?.reference?.split('/'))).filter((patientId) => !!patientId)
```

Comment:
I've already seen `.filter(Boolean)` used to do something similar. I find it easier to read, what do you think?